scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/spi/model/ModelValidatorContext.java

```diff
@@ -50,38 +51,46 @@
 
     private final Scheduler scheduler;
 
+    private final SchedulerSpaceInterface space;
+
     private String variableName;
 
     // container for job and task variables
     private SpELVariables spELVariables;
 
-    public ModelValidatorContext(StandardEvaluationContext context, Scheduler scheduler) {
+    public ModelValidatorContext(StandardEvaluationContext context, Scheduler scheduler,
```

Comment:
Having two constructors for ModelValidatorContext, one w/ SchedulerSpaceInterface, the other w/o this could have dwarfed your PR !

Reply:
Done.